packages/chart/src/components/EditorPanel.jsx

```diff
@@ -207,6 +207,95 @@ const DataSuppression = memo(({ config, updateConfig, data }) => {
     </>
   )
 })
+const DataPreliminary = memo(({ config, updateConfig, data }) => {
```

Comment:
Lets use the same naming convention of <PreliminaryData> for this component since that's whats we're calling it throughout the project.